### ISSUE TITLE
Opening R Project & executing terminal git commands causing 100% CPU usage in raven

### ISSUE THREAD

**Issue by sarahtanja** [RobertsLab/resources#1998]
On raven, I recently migrated my [coral-embryo-leachate](https://github.com/sarahtanja/coral-embryo-leachate) directory from 8TB_HDD_02 to 8TB_HDD_01.

When trying to open the R project in the new hard drive location I am running into some startup issue that is locking up raven's CPU usage  
![image](https://github.com/user-attachments/assets/14d4fa40-ffa2-4acd-a7cf-15eb89b706a0)

![image](https://github.com/user-attachments/assets/0e115b32-2573-43f9-a3da-2d18db93ecaf)

I can only get out of the lock-up when I close the Project...

But when I try to run any git command in terminal it locks up CPU usage again.
![image](https://github.com/user-attachments/assets/ab4277f5-4a65-4843-9cca-717e87c9e1d4)

**Comment by sr320** [comment]
This might not be easy to hear. but my recommendation is delete repo on raven, then pull repo from GitHub..

Related - if not already doing I recommend regular rsync from raven to
 gannet eg

```
rsync -avz  /home/shared/16TB_HDD_01/sr320/github/ \
--exclude='*.sam' \
--exclude='tmp*' \
--exclude='*gz_C_to_T.fastq' \
--exclude='*gz_G_to_A.fastq' \
sr320@gannet.fish.washington.edu:/volume2/web/seashell/bu-github/
```

**Comment by sarahtanja** [comment]
So I'd lose all the large files I've already generated 😢 Which admittedly is just:
- raw fastq.gz files uploaded from Azenta server
- raw MultiQC html, md5s
- qaqc fastp trimmed fastq.gz files
- clean MultiQC html, md5s

I haven't been able to push from raven to git for 2 weeks
My raw fastq.gz files are backed up to gannet and to ncbi 

If I delete everything and start again, is there enough space for me to actually generate the files I need for 63 RNA-Seq samples on `8TB_HDD_01`? 

Jackie offered to buy another hard drive to mount to raven if that would help resolve lack of space issues...

**Comment by sr320** [comment]
We can rsync to gannet before deleting

**Comment by kubu4** [comment]
> If I delete everything and start again, is there enough space for me to actually generate the files I need for 63 RNA-Seq samples on 8TB_HDD_01?

No on HDD01, but there's plenty of space on HDD02.

> Jackie offered to buy another hard drive to mount to raven if that would help resolve lack of space issues...

This would probably be ideal. 

She might already have one, though. Is anyone using the "old" computer I set up in her lab any more for RNA-seq analysis? There was (is?) an 8TB external HDD hooked up to it. If no one's using that, that would probably work...

**Comment by kubu4** [comment]
> Would you be able to take it and hook it up to raven?

Yes.

**Comment by kubu4** [comment]
> if it's as simple as unplugging it...

It's _mostly_ that simple. It will require a tiny bit of configuration. I should be able to tackle that after I get home this afternoon.

**Comment by sarahtanja** [comment]
In raven I ran the following command to `rsync` my directory to gannet

```
rsync -avz --progress /home/shared/8TB_HDD_01/stanja \
--exclude='*.sam' \
--exclude='tmp*' \
stanja@gannet.fish.washington.edu:/volume2/web/stanja
```
It looks like that command is still running, once it is complete I will delete the coral-embryo-leachate directory from raven

**Comment by kubu4** [comment]
Alrighty, the HDD is available on Raven here: `/media/4TB_JPG_ext`

However, I need to take a tiny bit of time to remove the existing data from my project(s) with Jackie. I'll let you know when it's cleared out.

**Comment by sarahtanja** [comment]
The `rsync` command is still running (been going for 16 hours)
![image](https://github.com/user-attachments/assets/44093ba9-9537-4fee-aab4-1c62c4e0dc99)
Is that normal for it to take this long?

**Comment by sarahtanja** [comment]
> You do not want to use that option in RStudio, though. The verbose output will cause RStudio to lock up.
.. ooof I did that in RStudio

**Comment by kubu4** [comment]
Oh, I see that now! 

You're probably going to need to kill that R session...

**Comment by sarahtanja** [comment]
I re-ran without the `--progress` option and backed everything up (and checked it) on gannet & then deleted the directory from 8TB_HDD_01

In summary:
- we decided there was no obvious fix (error may have been .cache files?, or some issue with moving the directory from one parent folder to another and having to reset global.options and PAT keys...?)
- we backed up the troublesome directory to gannet using `rsync` *without the --progress option*
- we deleted the directory from raven 8TB_HDD_01
- @kubu4 mounted a new 4TB external harddrive to migrate the project to

I'll open a new issue #2011 to follow up with the 4TB_JPG_ext clean out & project migration
